saleor/dashboard/product/views.py

```diff
@@ -610,3 +612,25 @@ def product_bulk_update(request):
             number=count) % {'count': count}
         messages.success(request, msg)
     return redirect('dashboard:product-list')
+
+
+@staff_member_required
+def ajax_variants_list(request):
+    def get_variant_label(variant):
+        return {
+            'id': variant.id,
+            'text': '%s, %s' % (
+                variant.display_product(), gross(variant.product.price))}
+
+    queryset = ProductVariant.objects.prefetch_related('product')
+    search_query = request.GET.get('q', '')
+    if search_query:
+        queryset = queryset.filter(
+            Q(sku__icontains=search_query) |
+            Q(name__icontains=search_query) |
+            Q(price_override__icontains=search_query) |
```

Comment:
I'm not sure if searching by price is very useful. Giving the ability to search by name and SKU should be enough in most cases.